### DISCUSSION TITLE
我clone项目后，yarn dev，本地对话报错：ConnectTimeoutError

### DISCUSSION THREAD

**Discussion by Friedrich-hue** [ChatGPTNextWeb/NextChat#2288] [Q&A]
通过查看控制台，会话请求的接口是：http://localhost:3000/api/openai/v1/chat/completions，
理论上本地运行也应该请求openai的官方接口吧，为什么会请求本地接口，本地没有开启服务怎么办？
应该改一下哪里的代码吗？

**Comment by Friedrich-hue** [comment]
应该跟下面这句代码有关：
const DEFAULT_OPENAI_URL =  getClientConfig()?.buildMode === "export" ? DEFAULT_API_HOST : "/api/openai/";
谁能告诉我为什么运行dev时，会默认路径为："/api/openai/"

**Reply by gym02** [reply]
我也遇到这个问题了，如何解决呢
